webpack.config.js

```diff
@@ -44,14 +44,7 @@ module.exports = {
     },
     disableHostCheck: true
   },
-  externals: [
-    "react",
-    "react-dom",
-    "react-router",
```

Comment:
react-router, react-router-dom, and single-spa-react should not be in the webpack externals. They get bundled into each project. Duplication of small dependencies like those is totally fine.